### PR TITLE
edge case fixes + playercard redo + eslint stuff

### PR DESCRIPTION
**Pull request by carteryu**
1. u can set no game now 
2. if ur not member of room, the button to leave room is no longer available
3. playercard updated to match chat
4. downgraded eslint to v5.x because v6 is super buggy with vscode (at least for me)

https://github.com/microsoft/vscode-eslint/issues/696 ^ 

pics/vid available upon request

### CODE REVIEW

File: civilapp/package.json [hunk 1]
```diff
@@ -40,7 +40,6 @@
     "react-native-splash-screen": "^3.2.0",
     "react-native-swipeout": "^2.3.6",
     "react-native-unimodules": "~0.5.2",
-    "react-native-vector-icons": "^6.6.0",
```
Comment (carteryu) [inline civilapp/package.json]: this was destroying me when i pod installed

File: civilapp/package.json [hunk 2]
```diff
@@ -52,7 +51,7 @@
     "@babel/core": "^7.0.0",
     "babel-eslint": "^10.0.3",
     "babel-preset-expo": "^6.0.0",
-    "eslint": "^6.7.2",
+    "eslint": "^5.8.0",
```
Comment (EphemeralX) [inline civilapp/package.json]: checking this in might make us downgrade, if we didn't have any issues.

File: civilapp/src/current/SportsTab.js [hunk 3]
```diff
@@ -67,6 +68,20 @@ class SportsTab extends React.Component {
     }
   }
 
+  componentDidMount() {
```
Comment (rauchp) [inline civilapp/src/current/SportsTab.js]: yo isn't this gonna fuck with my stuff?

File: civilapp/src/current/GameSelectScreen.js [hunk 4]
```diff
@@ -76,50 +71,82 @@ class GameSelectScreen extends React.Component {
...
+        disabled={selectedGameId === 'none'}
         saving={saving}
       />
     )
   }
 
+  renderDefaultGameBox = () => {
+    const { selectedGameId, doneLoading } = this.state
+    if (!doneLoading) {
+      return <></>
```
Comment (rauchp) [inline civilapp/src/current/GameSelectScreen.js]: did linter recommend this over null?

File: civilapp/package.json (hunk 2 above)
Comment (carteryu) [reply civilapp/package.json]: my thinking is if i had issues ppl onboarding will have issues too. see the link i posted regarding v6.0.x

File: civilapp/src/current/SportsTab.js (hunk 3 above)
Comment (carteryu) [reply civilapp/src/current/SportsTab.js]: linter was complaining it needed to be the first ting before anything else.  ill solve any . of ur conclits

**Pull request merged**